meta/include/actsvg/styles/defaults.hpp

```diff
@@ -15,123 +15,56 @@ namespace actsvg {
 namespace defaults {
 
 // Empty object for displaying
-static svg::object __e_object;
+extern svg::object __e_object;
 
 // Title style
-static style::font __t_font;
+extern style::font __t_font;
 
 // Sensitive surface section, including backside
-static style::fill __s_fill;
-static style::stroke __s_stroke;
+extern style::fill __s_fill;
+extern style::stroke __s_stroke;
```

Comment:
Beside my usual gripe about `extern`, is there really no way to make these objects `const`? It's really not great to have a bunch of non-const global objects. :frowning:

Reply:
I have made these const.

Reply:
Nevermind, these cannot easily be made const because the defaults have to be written to them. :(